tests/test_fastai_integration.py

```diff
@@ -17,11 +15,7 @@
 )
 
 from .testing_constants import ENDPOINT_STAGING, TOKEN, USER
-from .testing_utils import set_write_permission_and_retry
-
-
-def repo_name(id=uuid.uuid4().hex[:6]):
-    return "fastai-repo-{0}-{1}".format(id, int(time.time() * 10e3))
+from .testing_utils import repo_name, set_write_permission_and_retry
```

Comment:
Should we update the usage of `repo_name` in the test to pass a more explicit name to be consistent with before?

E.g. in https://github.com/huggingface/huggingface_hub/blob/af809b1f547e83b3080c45e95eecdb6c60c869cb/tests/test_fastai_integration.py#L92 change 

to
```python
REPO_NAME = repo_name("fastai_save") 
```

Reply:
Changed it 👍 

(in general I start to think it would be cool to have a fixture that generates the repo_id automatically based on the test name so that we don't have to bother about it anymore.)

Reply:
Yes, I fully agree!